rio_tiler/io/base.py

```diff
@@ -495,9 +497,11 @@ def tile(
         if expression:
             assets = self.parse_expression(expression, asset_as_band=asset_as_band)
 
+        assets = assets or self.default_assets
```

Comment:
what do you think about raising a Warning here when we use the default_assets? 

```python
if not assets and self.default_assets:
    warnings.warn(
        f"No assets/expression passed, defaults to {self.default_assets}", UserWarning
    )
    assets = self.default_assets
```

It adds couple lines 🤷‍♂️

Reply:
sounds good to me!